### PR TITLE
Mobile Menu

### PR DESCRIPTION
**Pull request by teddymaulana**

### CODE REVIEW

File: src/scss/compounds/_mobile-menu.scss [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+		transform: translateX(100%);
+		transition: transform 0.15s ease-in-out;
+		left: 0;
+		z-index: $zindex-sticky;
+		height: 100vh !important;
+		top: 0;
+
+		figure {
+			
+		}
```
Comment (foria) [inline src/scss/compounds/_mobile-menu.scss:130]: empty

File: src/scss/compounds/_mobile-menu.scss [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+	}
+
+	input:checked {
+		& ~ ul {
+			transform: translateX(0);
+			transition: transform 0.15s ease-in-out;
+		}
+	}
```
Comment (foria) [inline src/scss/compounds/_mobile-menu.scss:138]: there is a default function coming from bootstrap in order to open the mobile menu please check documentation

File: src/scss/compounds/_mobile-menu.scss [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+
+		.sni__range-tsw {
+			& ~ label h4 {
+				color: $range-tsw;
+			}
+		}
+
+		.active {
+
+		}
```
Comment (foria) [inline src/scss/compounds/_mobile-menu.scss:97]: empty

File: src/docs/compounds/mobile-menu.hbs [hunk 4]
```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
```
Comment (foria) [inline src/docs/compounds/mobile-menu.hbs:1]: I cannot see it

**Review by foria — CHANGES_REQUESTED** [review]

File: src/docs/compounds/mobile-menu.hbs (hunk 4 above)
Comment (teddymaulana) [reply src/docs/compounds/mobile-menu.hbs:1]: what cannot you see? what should be fixed?

File: src/scss/compounds/_mobile-menu.scss (hunk 2 above)
Comment (teddymaulana) [reply src/scss/compounds/_mobile-menu.scss:130]: this is not to open the mobile menu, this is to open submenu sequence, the function to open the mobile menu will be in the header later when integrate this branch

Screen Shot 2021-06-28 at 11 23 13

File: src/partials/compounds/mobile-menu.hbs [hunk 5]
```diff
@@ -0,0 +1,276 @@
+<nav class="mobile-nav position-fixed d-lg-none sni sni__times show">
+	<ul id="mobileMenu" class="mobile-nav__menu h-100 col-12 bg-white list-unstyled p-0">
+		<li class="d-flex justify-content-center w-100">
+			<a href="#" class="navbar-brand mx-auto sni sni__ss text-secondary"></a>
+		</li>
+		<li class="px-3 py-2 sni sni__chevron-next align-items-center w-100">
+			<input type="radio" name="menu" class="d-none" id="products">
+			<label class="position-relative m-0 w-100" for="products">Products</label>
+			<ul class="mobile-nav__menu-child position-absolute bg-white col-12 list-unstyled p-0 h-100">
+				<input type="radio" name="menu" id="back-to-menu-products" class="d-none">
```
Comment (foria) [inline src/partials/compounds/mobile-menu.hbs:10]: please check the position of the Back button, the position between the 2nd and 3rd menu level is different, I think the position in the 3rd level is the one correct

**Review by foria — CHANGES_REQUESTED** [review]

**Review by foria — APPROVED** [review]

**Pull request merged**